src/Ant/Button.elm

```diff
@@ -296,10 +338,10 @@ toHtml (Button options label) =
         attributes =
             case options.onClick of
                 Just msg ->
-                    [ A.disabled options.disabled, Events.onClick msg, css <| cursorHoverStyles :: combinedButtonStyles ]
+                    [ A.class "elm-antd__animated_btn", A.disabled options.disabled, Events.onClick msg, css <| cursorHoverStyles :: combinedButtonStyles ]
 
                 Nothing ->
-                    [ A.disabled options.disabled, css <| cursorHoverStyles :: combinedButtonStyles ]
+                    [ A.class "elm-antd__animated_btn", A.disabled options.disabled, css <| cursorHoverStyles :: combinedButtonStyles ]
```

Comment:
Could u please refactor this section? 

```elm
baseAttributes =
    [ A.class "elm-antd__animated_btn"
    , A.disabled options.disabled
    , Events.onClick msg
    , css <| cursorHoverStyles :: combinedButtonStyles 
    ]

attributes =
    case options.onClick of
        Just msg ->
            Events.onClick msg :: baseAttributes
        
        Nothing ->
            baseAttributes 
```

Don't forget to run `scripts/format-elm-files.sh` :)

Reply:
done. 

I am using elm-format so don't think I need to run format script :)